### ISSUE TITLE
Change the domain name creation process on Android

### ISSUE THREAD

**Issue by adamradocz** [BOINC/boinc#2846]
**Describe the problem**
The Boinc Client generates a domain name like `android_xxxxxxxx` on Android devices. It is really hard to track on the project's website  which device how much point generated when you have multiple devices.

**Describe the solution you'd like**
Change the domain name generation procedure to `Manufacturer - Model`, which is more human readable and recognizable then a random string.

**Additional context**
The `boinc/client/hostinfo_network.cpp` file have to be modified.

**Comment by AenBleidd** [comment]
@adamradocz, this could be not enough in the case when user has several devices with the same Manufacturer and Model.
Also there is a request for the same problem with different approach: #2721

**Comment by adamradocz** [comment]
Yes, I'm aware of #2721 issue. I think this, or similar approach should be the standard of the device's domain name creation. After that, the user still should be able to modifí it if it wants. BTW, if the user can modify the device name, he or she still can add the same name for two different devices.

And what about the `Manufacturer - Model - ID` approach?

**Comment by AenBleidd** [comment]
> I think this, or similar approach should be the standard of the device's domain name creation. 

Agree.

> BTW, if the user can modify the device name, he or she still can add the same name for two different devices.

Definitely, but if there is no way to modify device name manually then it will be confusing for user to see two devices with the same name because it would be too difficult to identify each device.

> And what about the `Manufacturer - Model - ID` approach?

Looks better for me and definitely better than current behaviour

**Comment by JuhaSointusalo** [comment]
Here's a stupid question from someone who doesn't run BOINC on Android. Doesn't the host list already contain extra information about Android devices? That extra information is only shown to the owner of the hosts so I can't go and look myself.

**Comment by AenBleidd** [comment]
Sorry about Russian language:
![image](https://user-images.githubusercontent.com/3234209/49044078-7c449e80-f1d5-11e8-9db9-25e730586369.png)

**Comment by AenBleidd** [comment]
First, third and fourth are Android devices
![image](https://user-images.githubusercontent.com/3234209/49044172-ad24d380-f1d5-11e8-9566-20f62b5e21d5.png)

**Comment by JuhaSointusalo** [comment]
@AenBleidd Thanks. So there is some information. Of course it still doesn't help if you have multiple devices of the same model.

**Comment by AenBleidd** [comment]
Yes, there is some information but if you have several same devices they will show equal or almost equal information because they have same hardware and probably the same version of software. So device name is the only way to identify definite device
